### PR TITLE
Correct the example in Node.p.previousSibling doc

### LINKED ISSUE #3377: Issue with "Node.previousSibling": Example refers to text nodes, not element nodes
MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Node/previousSibling

#### What information was incorrect, unhelpful, or incomplete?
In both `console.log`s in the Example, the `previousSibling` property actually points to a text node, not the previous image element.

#### Specific section or headline?
Example

#### Did you test this? If so, how?
I tested this by saving the example as an HTML file and examining it in Chrome DevTools.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/api/node/previoussibling`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Node/previousSibling
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/node/previoussibling/index.html
* Last commit: https://github.com/mdn/content/commit/63c69f3c933acbea9bfb24d359a7d2ffdb3c0bea
* Document last modified: 2021-02-19T19:51:22.000Z

</details>

### PR DESCRIPTION
**Pull request by longm89**
> What was wrong/why is this fix needed? (quick summary only)
The img elements are on separate lines, so there are text nodes between them. The expected results from console.log are then wrong.
> MDN URL of main page changed
https://developer.mozilla.org/en-US/docs/Web/API/Node/previousSibling
> Issue number (if there is an associated issue)
Fixes #3377
> Anything else that could help us review it
I added the correct expected results for console.log of the previous example and another example where I put the three img elements next to each other.

### CODE REVIEW

File: files/en-us/web/api/node/previoussibling/index.html [hunk 1]
```diff
@@ -21,15 +21,30 @@ <h2 id="Syntax">Syntax</h2>
 <pre class="brush: js"><var>previousNode</var> = <em>node</em>.previousSibling;
 </pre>
 
-<h2 id="Example">Example</h2>
-
-<pre class="brush: html">&lt;img id="b0"&gt;
-&lt;img id="b1"&gt;
-&lt;img id="b2"&gt;</pre>
+<h2 id="Examples">Examples</h2>
```
Comment (sideshowbarker) [inline files/en-us/web/api/node/previoussibling/index.html:24]:
```suggestion
<h2 id="Examples">Examples</h2>

<p>The following examples demonstrate how <code>previousSibling</code> works with and without text nodes mixed in with elements.</p>

```

File: files/en-us/web/api/node/previoussibling/index.html [hunk 2]
```diff
@@ -21,15 +21,30 @@ <h2 id="Syntax">Syntax</h2>
...
+<h3>First example</h3>
+<pre class="brush: html">&lt;img id="b0"&gt;&lt;img id="b1"&gt;&lt;img id="b2"&gt;</pre>
 
 <pre class="brush:js">console.log(document.getElementById("b1").previousSibling); // &lt;img id="b0"&gt;
 console.log(document.getElementById("b2").previousSibling.id); // "b1"
 </pre>
+<p>In this example, we put the img elements next to each other, so that there's no white space between them. 
+  See the next example and the Notes below for more detail.</p>
```
Comment (sideshowbarker) [inline files/en-us/web/api/node/previoussibling/index.html:32]:
```suggestion
```

File: files/en-us/web/api/node/previoussibling/index.html [hunk 3]
```diff
@@ -21,15 +21,30 @@ <h2 id="Syntax">Syntax</h2>
 <pre class="brush: js"><var>previousNode</var> = <em>node</em>.previousSibling;
 </pre>
 
-<h2 id="Example">Example</h2>
-
-<pre class="brush: html">&lt;img id="b0"&gt;
-&lt;img id="b1"&gt;
-&lt;img id="b2"&gt;</pre>
+<h2 id="Examples">Examples</h2>
+<h3>First example</h3>
```
Comment (sideshowbarker) [inline files/en-us/web/api/node/previoussibling/index.html:25]:
```suggestion
<h3>First example</h3>

<p>In this example, we have a series of <code>img</code> elements directly adjacent to each other, with no white space between them.</p>

```

File: files/en-us/web/api/node/previoussibling/index.html [hunk 4]
```diff
@@ -21,15 +21,30 @@ <h2 id="Syntax">Syntax</h2>
...
+</pre>
+
+<pre class="brush:js">
+    console.log(document.getElementById("b1").previousSibling); // #text
+    console.log(document.getElementById("b1").previousSibling.previousSibling); // &lt;img id="b0"&gt;
+    console.log(document.getElementById("b2").previousSibling.previousSibling); // &lt;img id="b1"&gt;
+    console.log(document.getElementById("b2").previousSibling); // #text
+    console.log(document.getElementById("b2").previousSibling.id); // undefined
+</pre>
+<p>In this example, there are text nodes between the img elements.</p>
```
Comment (sideshowbarker) [inline files/en-us/web/api/node/previoussibling/index.html:47]:
```suggestion
```

File: files/en-us/web/api/node/previoussibling/index.html [hunk 5]
```diff
@@ -21,15 +21,30 @@ <h2 id="Syntax">Syntax</h2>
...
+<h2 id="Examples">Examples</h2>
+<h3>First example</h3>
+<pre class="brush: html">&lt;img id="b0"&gt;&lt;img id="b1"&gt;&lt;img id="b2"&gt;</pre>
 
 <pre class="brush:js">console.log(document.getElementById("b1").previousSibling); // &lt;img id="b0"&gt;
 console.log(document.getElementById("b2").previousSibling.id); // "b1"
 </pre>
+<p>In this example, we put the img elements next to each other, so that there's no white space between them. 
+  See the next example and the Notes below for more detail.</p>
+<h3>Second example</h3>
```
Comment (sideshowbarker) [inline files/en-us/web/api/node/previoussibling/index.html:33]:
```suggestion
<h3>Second example</h3>

<p>In this example, there are whitespace text nodes (line breaks) between the <code>img</code> elements.</p>

```

File: files/en-us/web/api/node/previoussibling/index.html [hunk 6]
```diff
@@ -21,15 +21,30 @@ <h2 id="Syntax">Syntax</h2>
...
+<pre class="brush: html">
+  &lt;img id="b0"&gt;
+  &lt;img id="b1"&gt;
+  &lt;img id="b2"&gt;
```
Comment (sideshowbarker) [inline files/en-us/web/api/node/previoussibling/index.html:37]:
```suggestion
&lt;img id="b0"&gt;
&lt;img id="b1"&gt;
&lt;img id="b2"&gt;
```

File: files/en-us/web/api/node/previoussibling/index.html [hunk 7]
```diff
@@ -21,15 +21,30 @@ <h2 id="Syntax">Syntax</h2>
...
+</pre>
+
+<pre class="brush:js">
+    console.log(document.getElementById("b1").previousSibling); // #text
+    console.log(document.getElementById("b1").previousSibling.previousSibling); // &lt;img id="b0"&gt;
+    console.log(document.getElementById("b2").previousSibling.previousSibling); // &lt;img id="b1"&gt;
+    console.log(document.getElementById("b2").previousSibling); // #text
+    console.log(document.getElementById("b2").previousSibling.id); // undefined
```
Comment (sideshowbarker) [inline files/en-us/web/api/node/previoussibling/index.html:45]:
```suggestion
console.log(document.getElementById("b1").previousSibling); // #text
console.log(document.getElementById("b1").previousSibling.previousSibling); // &lt;img id="b0"&gt;
console.log(document.getElementById("b2").previousSibling.previousSibling); // &lt;img id="b1"&gt;
console.log(document.getElementById("b2").previousSibling); // #text
console.log(document.getElementById("b2").previousSibling.id); // undefined
```

File: files/en-us/web/api/node/previoussibling/index.html [hunk 8]
```diff
@@ -41,7 +56,10 @@ <h2 id="Notes">Notes</h2>
     or <a href="/en-US/docs/Web/API/Node/previousSibling"
       title="The Node.previousSibling read-only property returns the node immediately preceding the specified one in its parent's childNodes list, or null if the specified node is the first in that list."><code>Node.previousSibling</code></a>
     may refer to a whitespace text node rather than the actual element the author intended
-    to get.</p>
+    to get. One could use <a
+    href="/en-US/docs/Web/API/Element/previousElementSibling"><code><strong>Element.previousElementSibling</strong></code></a> to get an element node.
+  
+  </p>
```
Comment (sideshowbarker) [inline files/en-us/web/api/node/previoussibling/index.html:62]:
```suggestion
    to get. You can use <code><a href="/en-US/docs/Web/API/Element/previousElementSibling">previousElementSibling</a></code> to get the previous element node (skipping text nodes and any other non-element nodes).</p>
```

**Review by sideshowbarker — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/node/previoussibling/index.html [hunk 9]
```diff
@@ -21,15 +21,30 @@ <h2 id="Syntax">Syntax</h2>
...
-
-<pre class="brush: html">&lt;img id="b0"&gt;
-&lt;img id="b1"&gt;
-&lt;img id="b2"&gt;</pre>
+<h2 id="Examples">Examples</h2>
+<h3>First example</h3>
+<pre class="brush: html">&lt;img id="b0"&gt;&lt;img id="b1"&gt;&lt;img id="b2"&gt;</pre>
 
 <pre class="brush:js">console.log(document.getElementById("b1").previousSibling); // &lt;img id="b0"&gt;
 console.log(document.getElementById("b2").previousSibling.id); // "b1"
```
Comment (sideshowbarker) [inline files/en-us/web/api/node/previoussibling/index.html:29]:
```suggestion
<pre class="brush:js">
document.getElementById("b1").previousSibling;    // &lt;img id="b0"&gt;
document.getElementById("b2").previousSibling.id; // "b1"
```

**Review by sideshowbarker — APPROVED** [review]

**Comment by sideshowbarker** [comment]
Long, thanks much, and congrats on landing your first docs change here — welcome aboard 🎉

**Comment by longm89** [comment]
@sideshowbarker Thanks a lot for your correction. Have a good day!

**Pull request merged**